schema/banner.xsd

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  targetNamespace="http://epics.isis.rl.ac.uk/schema/banner/1.0"
+  xmlns:banner="http://epics.isis.rl.ac.uk/schema/banner/1.0"
+  elementFormDefault="qualified">
+  <xs:element name="banner">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element minOccurs="0" maxOccurs="unbounded" ref="banner:item"/>
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="item">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element ref="banner:name"/>
+        <xs:element ref="banner:pv"/>
+        <xs:element ref="banner:local"/>
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="name" type="xs:string"/>
+  <xs:element name="pv" type="xs:string"/>
+  <xs:element name="local" type="xs:boolean"/>
+</xs:schema>
```

Comment:
As discussed, should the block server really be serving a banner? The banner is a UI element. The Block Server should only serve model data. How about "Core controls"? Naming is hard, here is a brain dump of my current thoughts, maybe you can improve on that initial suggestion:

- These are instrument based, so you could say e.g. "Core instrument controls". I wonder whether including "instrument" is a tautology though as all information from the block server concerns the instrument
- These aren't blocks
- They aren't really controls because we don't give a mechanism to write to them in the bump strip
- I like core because they're the essential values associated with control of a given instrument. They are highlighted permanently above all others
- We shouldn't call them PVs as that's implementation specific and so overly restrictive
- Values or data are too vague to be useful
- They are kind of monitors but "Core monitors" sounds weird and is ambiguous with beam monitors
- Diagnostics? "Core diagnostics" or "Instrument diagnostics" could work. They're not really diagnostic in nature though, they're more superficial and high-level than that.
- Information? Is that just as vague as values and data? "Core information" or "Core control information"? I still wonder if that's too vague, but the best I've come up with so far.